cli/src/commands/did.rs

```diff
@@ -326,9 +394,9 @@ pub mod tests {
     pub const DID_MY1: &'static str = "VsKV7grR1BUE29mG2Fm2kX";
     pub const VERKEY_MY1: &'static str = "GjZWsBLgZCR18aL468JAT7w9CZRiBnpxUPPgyQxh4voa";
 
-    pub const SEED_MY2: &'static str = "00000000000000000000000000000My2";
-    pub const DID_MY2: &'static str = "2PRyVHmkXQnQzJQKxHxnXC";
-    pub const VERKEY_MY2: &'static str = "kqa2HyagzfMAq42H5f9u3UMwnSBPQx2QfrSyXbUPxMn";
+//    pub const SEED_MY2: &'static str = "00000000000000000000000000000My2";
```

Comment:
If MY2 group is no longer required please remove it.

Reply:
I left it because of order (now we have 1, 3, 4..), so later may be reused. :)